src/app/public/modules/ag-grid/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from './ag-grid.service';
 export * from './ag-grid.module';
 export * from './ag-grid.service';
 export * from './types';
+export * from './ag-grid-wrapper.component';
```

Comment:
Does the consumer ever need access to the component? If not, we probably don't need to export it.